liquibase-standard/src/test/groovy/liquibase/util/NetUtilTest.groovy

```diff
@@ -1,9 +1,12 @@
 package liquibase.util
 
+import spock.lang.IgnoreIf
 import spock.lang.Specification
+import spock.util.environment.OperatingSystem
 
 class NetUtilTest extends Specification {
 
+    @IgnoreIf({ OperatingSystem.current.macOs })
```

Comment:
Hey @Mahoney,

I don't think this is needed. We were also experiencing an issue on MacOS some time ago, but that was fixed with #6524. 

What kind of error were you getting here to skip this test?

Reply:
I think I made that change before that fix was merged. Test passes for me now, so I have reverted that change.